### ISSUE TITLE
Stock Entry from finished Work Order values finished good as the square of the quantity produced.

### ISSUE THREAD

**Issue by martinhbramwell** [frappe/erpnext#24385]
## Description of the issue

Please see full details in the forum post : 

[Production final valuations are the **square** of the # of items produced!?!?!](https://discuss.erpnext.com/t/production-final-valuations-are-the-square-of-the-of-items-produced/69745)

In a nutshell to produce 5 litres of finished product (Chemical D):

- 5 each of 4 raw material items (Labelled bottle and Chemicals A, B & C)
- Raw materials: $10.48 ($52.40 for 5)
- Operating costs: $8 ($40.00 for 5)
- Total: $18:48 ($92.40 for 5)

What I actually see in the Stock Entry generated from the finished Work Order is:

- Total Incoming Value: $ 302.00
- Total Outgoing Value: $ 52.40
- Total Value Difference (Out - In): $ 249.60

The only way I can see to get $302 incoming is by squaring!! the number of items, multiplying by raw materials costs then adding operating costs:

- 5 x 5 x $10.48 = $262
- $262 + 40 = $302

## Context information (for bug reports)

**Output of `bench version`**
``` python
erpdev@fossa:~/frappe-bench$ bench version
erpnext 13.0.0-beta.7
frappe 13.0.0-beta.9
erpdev@fossa:~/frappe-bench$ 
```

## Steps to reproduce the issue

1. Finish a Work Order that produced more than one item
2. Examine the valuations of the resulting draft Stock Entry
3.

### Observed result

See description and linked forum post mentioned above

### Expected result

See description and linked forum post mentioned above

### Stacktrace / full error message

```
N/A
```

## Additional information

OS version / distribution, `ERPNext` install method, etc.

``` python
erpdev@fossa:~/frappe-bench$ lsb_release -a
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal
erpdev@fossa:~/frappe-bench$ 
```

**Comment by martinhbramwell** [comment]
This is likely a testing failure
--

@vishdha
 The test rig  [test_stock_entry_for_material_transfer_for_manufacture.js](https://github.com/frappe/erpnext/blob/master/erpnext/stock/doctype/stock_entry/tests/test_stock_entry_for_material_transfer_for_manufacture.js#L12-L17) ...
```
stock/doctype/stock_entry/tests/test_stock_entry_for_material_transfer_for_manufacture.js
```

... looks like this:
```
	{items: [
		[
			{'item_code': 'Test Product 1'},
			{'qty': 1},
		]
	]},
```

Testing with a quantity of 1, will give the correct result even if the quantity is mistakenly used repeatedly in valuations.

**Comment by martinhbramwell** [comment]
The original problem seems to have been resolved in v13.0.0-beta.9

But [the test rig](https://github.com/frappe/erpnext/blob/v13.0.0-beta.10/erpnext/stock/doctype/stock_entry/tests/test_stock_entry_for_material_transfer_for_manufacture.js) has not been altered since 2017.
